htlcswitch/link.go

```diff
@@ -502,11 +501,11 @@ func (l *channelLink) Start() error {
 // NOTE: Part of the ChannelLink interface.
 func (l *channelLink) Stop() {
 	if !atomic.CompareAndSwapInt32(&l.shutdown, 0, 1) {
-		log.Warnf("channel link(%v): already stopped", l)
+		l.log.Warn("already stopped")
```

Comment:
Looks like the logging changes can perfectly fine go in its own PR separate from the far more critical commit sig changes.

Reply:
Separate prs created and linked.